app/controllers/users/backup_code_setup_controller.rb

```diff
@@ -25,11 +25,7 @@ def edit; end
 
     def continue
       flash[:success] = t('notices.backup_codes_configured')
-      next_mfa_setup_for_user = user_session.dig(
-        :selected_mfa_options,
-        determine_next_mfa_selection,
-      )
-      redirect_to user_next_authentication_setup_path(next_mfa_setup_for_user) ||
+      redirect_to next_setup_path ||
                   after_mfa_setup_path
```

Comment:
Personal style suggestion
```suggestion
      redirect_to next_setup_path || after_mfa_setup_path
```